app/routes/index.js

```diff
@@ -9,6 +9,18 @@ export default Ember.Route.extend({
         page: { refreshModel: true }
     },
 
+    beforeModel() {
```

Comment:
This same `beforeModel` is in `preprint.provider`, so it's run for every route in our app. Could we put it in the application route instead of having it twice?

Reply:
I'm going to refactor some stuff then come back to this.